lib/elixir/lib/kernel.ex

```diff
@@ -556,23 +558,104 @@ defmodule Kernel do
 
   @doc """
   Computes the remainder of an integer division.
+  `rem/2` uses truncated division, which means that 
```

Comment:
We should leave a blank line before this line as the first line of a `@doc` is used as the summary for the function in the module's function list.